guides/common/modules/ref_managing-packages-on-the-base-operating-system.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Managing_Packages_on_the_Base_Operating_System_{context}"]
+= Managing packages on the base operating system of {ProjectServer} or {SmartProxyServer}
+
+{Project} provides the `{foreman-maintain} packages` command to install and update packages on the base operating system of your {ProjectServer} or {SmartProxyServers}.
+
+[WARNING]
+====
+Installing and updating individual packages with `yum` is not supported.
+While {Project} provides the `{foreman-maintain} packages unlock` command to bypass `{foreman-maintain} packages` and enable package management with `yum`, this can lead to package inconsistencies and compromise the stability of the system.
+Always use `{foreman-maintain} packages` to manage packages.
+====
+
+After installing or updating a package on the base operating system, it is required to run `{foreman-installer}`.
+The `{foreman-maintain} packages` command automatically runs `{foreman-installer}` after you use it to install or update a package.
+Note that this results in restarting affected services on the base operating system.
```

Comment:
I think the installer doesn't manage services on the base OS but really the Satellite services.

Reply:
How about:
```suggestion
Note that this might result in restarts of services when their configuration changes due to the updates.
```
(completely avoiding OS vs Satellite)

Reply:
Applied, slightly modified.